src/EnergyPlus/RefrigeratedCase.cc

```diff
@@ -12406,96 +12400,96 @@ void SimulateDetailedTransRefrigSystems(EnergyPlusData &state)
     auto &GasCooler(state.dataRefrigCase->GasCooler);
     auto &WalkIn(state.dataRefrigCase->WalkIn);
 
-    int LocalTimeStep = state.dataGlobal->TimeStepZone;
+    Real64 LocalTimeStep = state.dataGlobal->TimeStepZone;
```

Comment:
Shouldn't this cause diffs?

Reply:
Technically it could have, but looks like that variable was only used inside a loop over the transcritical refrigeration systems and only if (not using system time step and not warmup), so maybe we just don't hit anything like that in the example files?

Interesting, we do actually.  It hits the pipe/total energy calculation lines 288 times so they are in at least one example file.  The values must be calculated at zero, or we are lucking out on the integer and float being essentially the same. Coverage results here, check out line 12609-12612: http://energyplus.s3-website-us-east-1.amazonaws.com/coverage/2023-02-23-EnergyPlus-8126b3bb85-x86_64-gcc-7.5ICD/EnergyPlus/RefrigeratedCase.cc.gcov.html

Reply:
That must mean something else in those equations evaluate to 0, and it's not SecInHour. I wonder if those reports show non-zero values? Something is strange.

Reply:
I didn't find any example files with outputs for Suction Pipe Heat Gain Energy. That answers that.